### DISCUSSION TITLE
有没有考虑过加入playground就像vue与elementplus一样？

### DISCUSSION THREAD

**Discussion by baiwusanyu-c** [arco-design/arco-design-vue#1828] [Q&A] [answered]
有没有考虑过加入 `playground` 就像 `vue` 与 `elementplus` 一样？在给 `vue` 和 `elementplus` 做pr或者 `issue` 时，发现这东西很方便能够快速复现一个问题环境，还能看到 `vue` 编译后结果，`arco-design-vue`  有考虑加入这个 `playground` 吗

**Comment by hehehai** [comment, marked as answer]
@baiwusanyu-c hi, 可以试试这个 https://github.com/hehehai/arco-vue-playground

**Reply by baiwusanyu-c** [reply]
This is exactly what I need. I don't know why it was not added to the project, but at least I hope it can be added to the relevant documents, so that many people don't know where to use Playground when they mention issue
